metadata-ingestion/src/datahub/ingestion/source/lookml.py

```diff
@@ -66,6 +68,12 @@ class LookMLSourceConfig(ConfigModel):
     view_pattern: AllowDenyPattern = AllowDenyPattern.allow_all()
     env: str = builder.DEFAULT_ENV
     parse_table_names_from_sql: bool = False
+    # This source can be configured to use a custom SQL parser.
```

Comment:
Unfortunately it violates DRY, but we are also maintaining configuration documentation for each sink under sink_docs. 

Could you update the lookml documentation? 
https://github.com/linkedin/datahub/blob/master/metadata-ingestion/source_docs/lookml.md

Reply:
@shirshanka  Fixed!